src/TritonCTS/src/DbWrapper.cpp

```diff
@@ -189,9 +226,10 @@ void DbWrapper::computeITermPosition(odb::dbITerm* term, DBU &x, DBU &y) const {
                         ++numShapes;                 
                 }
         }
-
-        x /= numShapes;
-        y /= numShapes;
+       if (numShapes > 0){
```

Comment:
Why doesn't this use iterm::getAvgXY instead of grovelling around in the shapes?